### PR TITLE
Standardize node names and images in kne topologies

### PR DESCRIPTION
**Pull request by greg-dennis**

### CODE REVIEW

**Comment by octpetre** [comment]
I'm getting errors while trying to run a test after deploying the topology with these changes
```
opetre@psodev1-ubserv:~/featureprofiles$ go test -v feature/interface/staticarp/otg_tests/static_arp_test/*.go -kne-config ~/arista.kne.yml -testbed ~/featureprofiles/topologies/atedut_2.testbed

*** Reserving the testbed...

F0203 21:20:09.935961   20833 ondatra.go:51] error executing command /home/opetre/go/bin/kne_cli topology service /home/opetre/featureprofiles/topologies/kne/arista_ixia_lag.textproto: I0203 21:20:09.904762   20908 topo.go:118] Trying in-cluster configuration
I0203 21:20:09.905724   20908 topo.go:121] Falling back to kubeconfig: "/home/opetre/.kube/config"
I0203 21:20:09.908120   20908 topo.go:254] Adding Link: otg:eth1 dut:eth1
I0203 21:20:09.908142   20908 topo.go:254] Adding Link: dut:eth2 otg:eth2
I0203 21:20:09.908150   20908 topo.go:254] Adding Link: arista:eth3 otg:eth3
Error: service <topology>: failed to load topology: invalid topology: missing node "arista"
Log files can be found in:
    /tmp/kne_cli.psodev1-ubserv.opetre.log.INFO.20230203-212009.20908
: exit status 1
FAIL    command-line-arguments  0.564s
FAIL
opetre@psodev1-ubserv:~/featureprofiles$

**Comment by greg-dennis** [comment]
Thanks @octpetre, I missed a few. I'm testing all the topologies again now

File: topologies/kne/arista_ceos_otg.textproto [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: "ixia-c"
 nodes: {
-    name: "ate"
+    name: "otg"
```
Comment (liulk) [inline topologies/kne/arista_ceos_otg.textproto:3]: I think OTG tests still use `ondatra.ATE(t, "ate")` not `ondatra.ATE(t, "otg")`. Would this cause problem?

https://github.com/openconfig/featureprofiles/blob/317a6776ef8844d31bcece10c4d50621e6814bb0/feature/interface/singleton/otg_tests/singleton_test/singleton_test.go#L517

Comment (greg-dennis) [reply topologies/kne/arista_ceos_otg.textproto:3]: No, because the Ondatra names are from the testbed, not the KNE topology

**Review by Ankur19 — APPROVED** [review]
LGTM!

**Pull request merged**